static/docs/user-guide/external-data/hdfs.md

```diff
@@ -0,0 +1,89 @@
+# External Data on HDFS
```

Comment:
let's remove all external data section for HDFS. We don't support it actually and it should be remove from DVC for now. cc @efiop can confirm that.

Reply:
Confirmed, we were not able to find a suitable workaround for external outputs. 🙁 So let's hide hdfs external deps and outs from docs for now. We'll probably remove them from dvc too very soon.

Reply:
> So let's hide hdfs external deps and outs from docs for now.

I am removing the section "External Data and Outputs". But it seems like "External Dependencies" should work. Please correct me if I am wrong.

Reply:
@dashohoxha even external dependencies do not work reliable unfortunately as far as I remember

Reply:
Removed.